src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/Regexp/patchedline/defaultContextConfig.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+          "https://checkstyle.org/dtds/configuration_1_3.dtd">
+<module name="Checker">
+    <module name="TreeWalker">
+        <module name="Regexp">
+            <property name="format" value="// This code is copyrighted\n// \(c\) MyCompany"/>
+        </module>
+
+        <module name="com.puppycrawl.tools.checkstyle.filters.SuppressionPatchXpathFilter">
+            <property name="file" value="${tp}/defaultContext.patch" />
+            <property name="strategy" value="patchedline" />
+            <property name="neverSuppressedChecks" value="RegexpCheck" />
```

Comment:
Why do you need this property for a patchedline strategy?

Reply:
same answer https://github.com/checkstyle/patch-filters/pull/207#discussion_r467540648